src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```diff
@@ -114,14 +114,16 @@ HTM_046=Fixed format item has no viewport defined.
 HTM_046_SUG=A viewport declaration is required for fixed format items.
 HTM_047=Html viewport is missing height and/or width.
 HTM_047_SUG=The viewport declaration must declare both width and height.
-HTM_048=SVG ViewBox is missing on fixed format document.
-HTM_048_SUG=A viewBox declaration is required for fixed format documents.
+HTM_048=SVG ViewBox and width and height are missing on fixed format document.
+HTM_048_SUG=A viewBox declaration or width and height are required for fixed format documents.
 HTM_049=Html element does not have an xmlns set to 'http://www.w3.org/1999/xhtml'.
 HTM_049_SUG=Add xmlns="http://www.w3.org/1999/xhtml" to the html element.
 HTM_050=Found epub:type="pagebreak" attribute in content document.
 HTM_051=Found Microdata semantic enrichments but no RDFa. EDUPUB recommends using RDFa Lite.
 HTM_052=The property 'region-based' is only allowed on nav elements in Data Navigation Documents.
 HTM_053=Found an external file link (file://) in file: '%1$s'.
+HTM_054=Only 'width' and 'height' were found for an SVG element.  SVG elements should include a 'viewBox' property. File: '%1$s'
+HTM_055=Width and Height properties should be in pixel sizes for fixed format documents. File: '%1$s'
```

Comment:
suggested rewording:
>The  'width' and 'height' attribute values should be set to absolute lengths using the pixel unit.

Reply:
It sort of sounds a bit too techy. I'd maybe go a little simpler with:

> The values of the 'height' and 'width' attributes should be specified in pixels.